users/views.py

```diff
@@ -1,3 +1,57 @@
-from django.shortcuts import render
+import json
+import jwt
+import requests
+from datetime     import datetime
 
-# Create your views here.
+from django.http  import JsonResponse
+from django.views import View
+
+from .models      import User
+from my_settings  import SECRET_KEY
+
+
+class KakaoSigninView(View):
+	def get(self, request):
+		try:
+			access_token = request.headers.get("Authorization")
+
+			if not access_token:
+				return JsonResponse({"message" : "NEED_TOKEN"}, status = 400)
+
+			profile_request = requests.get(
+				"https://kapi.kakao.com/v2/user/me", headers = {"Authorization" : f"Bearer {access_token}"}
+			)
```

Comment:
여기서는 두가지 피드백을 드릴 수 있을 것 같습니다.
1. 변수를 선언했을 때는 해당 변수에 담기는 값이 무엇인지 명확하게 작성할 것.
```suggestion
response = requests.get(
    "https://kapi.kakao.com/v2/user/me", 
    headers = {"Authorization" : f"Bearer {access_token}"}
)
```
2. requests 모듈을 사용할 때에는 반드시 `timeout` option을 걸어 주셔야합니다. 그렇지 않으면 요청을 보낸 서버로부터 응답이 올때까지 무기한 기다리는 문제가 발생하게 됩니다.